tests/test_actions.py

```diff
@@ -753,22 +755,26 @@ def test_not_performable(self) -> None:
         with pytest.raises(NotPerformable) as exc:
             SilentlyPerformable(None)  # type: ignore
 
-        assert str(exc.value) == ("SilentlyPerformable only works with Performable. "
-                                  "Use `Silently` instead.")
+        assert str(exc.value) == (
+            "SilentlyPerformable only works with Performable. "
+            "Use `Silently` instead."
+        )
 
     def test_not_answerable(self) -> None:
         with pytest.raises(NotAnswerable) as exc:
             SilentlyAnswerable(None)  # type: ignore
 
-        assert str(exc.value) == ("SilentlyAnswerable only works with Answerable. "
-                                  "Use `Silently` instead.")
+        assert str(exc.value) == (
+            "SilentlyAnswerable only works with Answerable. " "Use `Silently` instead."
+        )
```

Comment:
Minor thing but this really should be a single string.  (mildly concerning if Black did this)

Reply:
Hopefully they fix that.  They are usually really good about not "making it worse".